Makefile

```diff
@@ -0,0 +1,18 @@
+DOCKER := $(shell which docker)
+
+protoVer=0.13.1
```

Comment:
Can we use `0.14.0`? Related projects (e.g. yui-relayer and ibc-go) use this version.

Reply:
fixed